actions/add_update_repository_environment.yaml

```diff
@@ -0,0 +1,73 @@
+---
+name: add_update_repository_environment
+# https://docs.github.com/en/enterprise-server@3.2/rest/deployments/environments
+runner_type: python-script
+pack: github
+description: >
+  Add or update a repository environment.
+  Example:
+    st2 run github.add_update_repository_environment  organization="organization" owner="owner" repo="reponame" reviewers="< array of reviewers >" api_user="token_name" 
```

Comment:
I can't see a parameter organization in the yaml. It looks like in the python code, that owner is being passed as organisation to the get_team_id - so that parameters in the example and the yaml don't seem to match.
Also yaml says environment is required parameter with no default, yet its not used in the example.

Reply:
thanks! I have rewritten this and tested the call, it seems to be okay now :)

still pending to push the code though